extensions/amp-story/1.0/prerender-active-page.js

```diff
@@ -3,10 +3,9 @@ import {parseQueryString} from '#core/types/string/url';
 import {getWin} from '#core/window';
 
 /**
- * Returns true if the page should be prerendered (for being an active page or
- * first page).
- * @param {!AmpElement} pageElement
- * @return {boolean}
+ * Retrieves the page that should be the prerender page.
```

Comment:
The previous comment was more helpful, it is not necessarily the prerender page but is the active one, regardless of the visibilitystate